pkg/storage/chunk/gcp/bigtable_object_client.go

```diff
@@ -9,7 +9,7 @@ import (
 	otlog "github.com/opentracing/opentracing-go/log"
 	"github.com/pkg/errors"
 
-	"github.com/cortexproject/cortex/pkg/util/math"
+	"github.com/grafana/loki/pkg/util/math"
 
```

Comment:
One more here. Sorry for being picky, we would miss fixing it until we start working on it again.